src/Runner.js

```diff
@@ -175,20 +175,21 @@ export default class Runner extends EventEmitter {
     delete this.debugprocess;
   }
 
-  findMessageType(buf: Buffer): MessageType {
-    const str = buf.toString('utf8', 0, 58);
-    const lastCommitRegex = /No tests found related to files changed since ((last commit)|("[a-z0-9]+"))./;
-    if (lastCommitRegex.test(str)) {
-      return messageTypes.noTests;
-    }
-    if (/^\s*Watch Usage\b/.test(str)) {
-      return messageTypes.watchUsage;
-    }
 
-    if (str.trim().startsWith('Test results written to')) {
-      return messageTypes.testResults;
-    }
-    return messageTypes.unknown;
+  findMessageType(buf: Buffer): MessageType {
```

Comment:
I think it's extremely unlikely, but technically this still might not catch all the files. Because the buffer can be flushed partway, it's possible that strings are actually split over multiple messages. Anecdotally, I've noticed the test output can be split mid-word. However, due to these messages usually being at the beginning/end of the test output, I don't think it'll be a problem.

Anyways, thanks for fixing this! I thought i was going insane trying to figure out why some of my projects would work but others wouldn't...

Reply:
you have a good point, I haven't seen these sentences split, but in theory, it could happen, especially if the output is large I guess... 

We could potentially buffer the sentence without EOL but that won't solve the problem when no-change message comes before the output-file-is-written message (now in a separate chuck)...

I am hesitated to add more code for a theoretical scenario that we haven't seen much evidence yet... let's make a note here and come back to it once there is more evidence.

Reply:
Yea, I was just making a comment here, I think the code you have is perfectly fine, just noting for future reference in case something ever happens here!